_includes/layouts/base.njk

```diff
@@ -46,7 +46,42 @@
     <link rel="preconnect" href="/" crossorigin>
     <script async defer src="{{ "/js/min.js" | addHash }}"></script>
     {% if googleanalytics %}
-    <script async defer src="{{ "/js/cached.js" | addHash }}"></script>
+      <script async defer src="{{ "/js/cached.js" | addHash }}"></script>
+      {% if webvitals %}
+        <script defer src="{{ "/js/web-vitals.js" | addHash }}"></script>
+        <script csp-hash>
+          addEventListener('DOMContentLoaded', () => {
+            // Function taken from https://github.com/GoogleChrome/web-vitals#using-analyticsjs
```

Comment:
Inline scripts aren't optimized today. But come to think of it, that might be a good feature to add.

So, maybe OK to keep the comments and I can do that separately

Reply:
Ok. If you prefer I move this to an external file in the meantime, I can make that change.

Reply:
How about merging this code into the web-vitals.js and loading it async from main.js (you can add a preload here for performance reasons)?

Reply:
(That would eliminate the race condition mentioned above)

Reply:
Sounds good, I will go with this approach.